### PR TITLE
add option for nonzero sky

### PR DESCRIPTION
**Pull request by esheldon**
new keyword sky_n_sigma adds noise * sky_n_sigma to each image

added a test

### CODE REVIEW

**Comment by esheldon** [comment]
bump @beckermr

File: .github/workflows/shear_meas_tests.yml [hunk 1]
```diff
@@ -31,14 +31,15 @@ jobs:
           conda config --set always_yes yes
           conda install -q mamba
 
-          mamba install -q stackvana=0
+          # mamba install -q stackvana=0
+          mamba install stackvana=0.2021.32
```
Comment (beckermr) [inline .github/workflows/shear_meas_tests.yml:35]: This is not needed. I patched galsim so it works for us on v 2.2.6

File: .github/workflows/shear_meas_tests.yml [hunk 2]
```diff
@@ -31,14 +31,15 @@ jobs:
...
 
           mamba install -q \
             flake8 \
             pytest \
             numpy \
-            galsim \
-            numba \
+            "galsim>=2.3" \
```
Comment (beckermr) [inline .github/workflows/shear_meas_tests.yml:41]: This pin can be removed.

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -31,14 +31,15 @@ jobs:
           conda config --set always_yes yes
           conda install -q mamba
 
-          mamba install -q stackvana=0
+          # mamba install -q stackvana=0
+          mamba install stackvana=0.2021.32
```
Comment (beckermr) [inline .github/workflows/test.yml:35]: Same comment as above.

File: .github/workflows/test.yml [hunk 4]
```diff
@@ -31,14 +31,15 @@ jobs:
...
 
           mamba install -q \
             flake8 \
             pytest \
             numpy \
-            galsim \
-            numba \
+            "galsim>=2.3" \
```
Comment (beckermr) [inline .github/workflows/test.yml:41]: Same comment as above.

**Review by beckermr — CHANGES_REQUESTED** [review]
We can back out the pins in the CI scripts. Code looks good!

Sorry for the delay in getting to this one and thank you for the bump!

**Review by beckermr — APPROVED** [review]

**Comment by esheldon** [comment]
thanks for the review Matt

**Pull request merged**